torchreid/engine/image/am_softmax.py

```diff
@@ -377,4 +412,175 @@ def _apply_batch_transform(self, imgs, obj_ids):
             permuted_idx = torch.randperm(imgs.shape[0])
             imgs = lambd * imgs + (1 - lambd) * imgs[permuted_idx]
 
+        if self.aug_type == 'fmix':
```

Comment:
Since now we have *mix augmentation implemented, I'd ask to remove `self.batch_transform_cfg.enable ...` above in this function.